tests/libstorage/pvc.go

```diff
@@ -180,38 +182,62 @@ func createPVC(pvc *k8sv1.PersistentVolumeClaim, namespace string) *k8sv1.Persis
 	return createdPvc
 }
 
-func CreateFSPVC(name, namespace, size string, labels map[string]string) *k8sv1.PersistentVolumeClaim {
+func CreateFSPVC(name, namespace, size string) *k8sv1.PersistentVolumeClaim {
 	sc, _ := GetRWOFileSystemStorageClass()
-	pvc := NewPVC(name, size, sc)
-	volumeMode := k8sv1.PersistentVolumeFilesystem
-	pvc.Spec.VolumeMode = &volumeMode
-	if labels != nil && pvc.Labels == nil {
-		pvc.Labels = map[string]string{}
-	}
+	return createPVC(NewPVC(name, size, sc), namespace)
+}
 
+func CreateFSPVCWithDataVolume(name, namespace, size string, labels map[string]string) *k8sv1.PersistentVolumeClaim {
```

Comment:
I kinda agree with other reviewers that seems we’re hiding some behavior with the storage profile/dv creation. To other maintainers, this dv creation might look arbitrary since the reason to create it (minimum size enforcement) isn’t stated in comments or isn't obvious from the code. I’m not against merging the pr, but I think we might want to make this more explicit.

Reply:
I thought naming is enough, but I'll sure add a comment for clarification.